readthedocs/settings/base.py

```diff
@@ -31,13 +31,12 @@ class CommunityBaseSettings(Settings):
 
     # Debug settings
     DEBUG = True
-    TEMPLATE_DEBUG = DEBUG
     TASTYPIE_FULL_DEBUG = True
 
     # Domains and URLs
     PRODUCTION_DOMAIN = 'readthedocs.org'
     PUBLIC_DOMAIN = None
-    USE_SUBDOMAIN = False
+    USE_SUBDOMAIN = True
```

Comment:
Why you changed this default?

Reply:
@kakulukia Any feedback on this?

Reply:
this shouldnt have made it into the PR - its already reverted and github anlready maks this as outdated for me